pkg/cloudprovider/cloudapi/aws/aws_converters.go

```diff
@@ -45,28 +45,39 @@ func convertToIPPermissionPort(port *int, protocol *int) (*int64, *int64) {
 	return portVal, portVal
 }
 
-func convertToEc2IpRanges(ips []*net.IPNet, ruleHasGroups bool, description *string) []*ec2.IpRange {
-	var ipRanges []*ec2.IpRange
+// convertToEc2IpRanges convert the IPs to EC2 IPv4 and IPv6 ranges.
+func convertToEc2IpRanges(ips []*net.IPNet, ruleHasGroups bool, description *string) ([]*ec2.IpRange, []*ec2.Ipv6Range) {
+	var ipv4Ranges []*ec2.IpRange
+	var ipv6Ranges []*ec2.Ipv6Range
 	if len(ips) == 0 && !ruleHasGroups {
 		ipRange := &ec2.IpRange{
 			CidrIp:      aws.String("0.0.0.0/0"),
 			Description: description,
 		}
-		ipRanges = append(ipRanges, ipRange)
-		return ipRanges
+		ipv4Ranges = append(ipv4Ranges, ipRange)
+		return ipv4Ranges, ipv6Ranges
```

Comment:
This means allow all right? So shouldn't an IPv6 `::` range also be added?

Reply:
you are right.. let me add that